macros/src/main/scala/com/softwaremill/macwire/MacwireMacros.scala

```diff
@@ -33,7 +33,7 @@ object MacwireMacros {
             var newT: Tree = Select(New(Ident(targetTpe.typeSymbol)), termNames.CONSTRUCTOR)
 
             for {
-              targetConstructorParams <- targetConstructorParamLists if !targetConstructorParams.exists(_.isImplicit)
+              targetConstructorParams <- targetConstructorParamLists if !targetConstructorParams.headOption.exists(_.isImplicit)
```

Comment:
true, that looks shady :) looks like the first parameter can't be an implicit member